apps/docs-snippets/src/guide/cookbook/signing-transactions.test.ts

```diff
@@ -0,0 +1,122 @@
+import type { Provider, BN } from 'fuels';
+import {
+  WalletUnlocked,
+  Predicate,
+  BaseAssetId,
+  Script,
+  FunctionInvocationResult,
+  ScriptTransactionRequest,
+} from 'fuels';
+
+import {
+  DocSnippetProjectsEnum,
+  getDocsSnippetsForcProject,
+} from '../../../test/fixtures/forc-projects';
+import { getTestWallet } from '../../utils';
+
+/**
+ * @group node
+ */
+describe('Signing transactions', () => {
+  let sender: WalletUnlocked;
+  let receiver: WalletUnlocked;
+  let signer: WalletUnlocked;
+  let provider: Provider;
+  let gasPrice: BN;
+  const { abiContents: abiPredicate, binHexlified: binPredicate } = getDocsSnippetsForcProject(
+    DocSnippetProjectsEnum.PREDICATE_SIGNING
+  );
+  const { abiContents: abiScript, binHexlified: binScript } = getDocsSnippetsForcProject(
+    DocSnippetProjectsEnum.SCRIPT_SIGNING
+  );
+
+  beforeAll(async () => {
+    sender = await getTestWallet();
+    signer = WalletUnlocked.generate({
+      provider: sender.provider,
+    });
+
+    provider = sender.provider;
+    ({ minGasPrice: gasPrice } = provider.getGasConfig());
+  });
+
+  beforeEach(() => {
+    receiver = WalletUnlocked.generate({
+      provider: sender.provider,
+    });
+  });
+
+  it('creates a transfer with external signer [script]', async () => {
+    const amountToReceiver = 100;
+
+    // #region multiple-signers-2
+    // #import { Script, BaseAssetId };
+
+    // Create invocation scope
+    const script = new Script(binScript, abiScript, sender);
+    const scope = script.functions
+      .main(signer.address.toB256())
+      .txParams({ gasPrice, gasLimit: 10_000 });
```

Comment:
This is an awesome PR. Huge wins here! Still reviewing it, but I noticed this. You don't need to specify these values anymore after #1708

Reply:
Interestingly, both tests fail if I remove `txParams` 🤔

Reply:
Oh interesting, what does it say?

Reply:
Script throws `OutOfGas` and predicate `Transaction is not inserted. The gas price is too low`.

Reply:
This has been resolved by @Torres-ssf in #1926